src/xdocs/checks.xml

```diff
@@ -791,6 +791,17 @@
               when the <code>@inheritDoc</code> javadoc tag is present.
             </td>
           </tr>
+          <tr>
+            <td>
+              <a href="missingnullcaseinswitch.html#MissingNullCaseInSwitch">
```

Comment:
This URL is wrong. This is why it appears in linkcheck. The suppression added is also wrong.

Make sure you verify the site and links you are adding.

Reply:
done